masterlist.yaml

```
@@ -16616,9 +16616,7 @@ plugins:
       - crc: 0x620CEAB4
         util: 'SSEEdit v4.0.4b'
   - name: 'RCBE - Thwack Consistency Patch.esp'
-    msg:
-      - <<: *useInstead
-        subs: [ '[Thwack - A Creation Club Integration Mod (Thwack! - Rare Curios - Bolts Expanded.esp)](https://www.nexusmods.com/skyrimspecialedition/mods/42653/)' ]
+    msg: [ *obsolete ]
```

`obsolete` implies that there's either a newer version of the patch or a newer version of the mod that doesn't require a patch. `patchOutdated` is more appropriate here:

```suggestion
    msg: [ *patchOutdated ]
```